packages/e2e-tests/src/specs/editor/media/insert3PMedia.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { createNewStory } from '@web-stories-wp/e2e-test-utils';
+
+describe('Inserting 3P Media', () => {
+  it('should dismiss message', async () => {
+    await createNewStory();
+
+    await expect(page).not.toMatchElement('[data-testid="FrameElement"]');
+    await expect(page).toClick('#library-tab-media3p');
+    await expect(page).toClick('button', { text: 'Dismiss' });
```

Comment:
Nit: this only works if tests are executed in order and if this is the first media3p test to run. So not 100% bulletproof.